ui/admin/app/routes/scopes/scope/credential-stores/credential-store/credential-libraries.js

```diff
@@ -3,19 +3,28 @@ import { action } from '@ember/object';
 import loading from 'ember-loading/decorator';
 import { confirm } from 'core/decorators/confirm';
 import { notifySuccess, notifyError } from 'core/decorators/notify';
+import { inject as service } from '@ember/service';
 
 export default class ScopesScopeCredentialStoresCredentialStoreCredentialLibrariesRoute extends Route {
   // =methods
+  @service can;
 
   /**
    * Loads all credential libraries under the current credential store.
    * @return {Promise{[CredentialLibraryModel]}}
    */
   model() {
-    const { id: credential_store_id } = this.modelFor(
+    const scope = this.modelFor(
```

Comment:
This is getting a credential store, not a scope, so I'd just tweak the constant name to match.